backend/CircleForms/Models/Posts/Answer.cs

```diff
@@ -1,22 +1,35 @@
 ﻿using System.Collections.Generic;
+using System.Threading.Tasks;
 using CircleForms.Models.Posts.Questions.Submissions;
-using MongoDB.Bson.Serialization.Attributes;
+using CircleForms.Models.Users;
 using MongoDB.Entities;
 using Newtonsoft.Json;
 
 namespace CircleForms.Models.Posts;
 
-public class Answer : IEntity
+public class Answer : Entity
 {
     [JsonProperty("answers")]
     public List<Submission> Submissions { get; set; }
 
-    [BsonId]
-    [JsonProperty("user_id")]
-    public string ID { get; set; }
+    [JsonProperty("user")]
+    public One<User> User { get; set; }
```

Comment:
Why can't we do the same for forms?

Reply:
because I remembered about it 1 day ago

Reply:
I think it might be worth doing in a separate PR